include/aspect/utilities.h

```diff
@@ -1060,6 +1066,41 @@ namespace aspect
      * entry in the list must match one of the allowed operations.
      */
     std::vector<Operator> create_model_operator_list(const std::vector<std::string> &operator_names);
+
+    /**
+    * A class that represents a point in a chosen coordinate system.
```

Comment:
you are not choosing the coordinate system here

Reply:
I think what @tjhei wants to say that you should use the wording "in a given coordinate system that may or may not be Cartesian".